libraries/textcomposer/impl/src/main/kotlin/io/element/android/libraries/textcomposer/TextComposer.kt

```diff
@@ -255,7 +257,8 @@ private fun TextInput(
                     start = 12.dp.applyScaleUp(),
                     end = 42.dp.applyScaleUp()
                 )
-            ),
+            )
+            .testTag(TestTags.richTextEditor),
```

Comment:
It looks like adding size changes the size of the component... I am on it.

Reply:
3c81f12f932f53ece0066078673c1ab30b0db33c fixes, running the test again so that there is no regression elsewhere due to this change (or maybe other bugfix (?)).